### ISSUE TITLE
pyscreeze.screenshot makes a full desktop screenshot instead of region

### ISSUE THREAD

**Issue by mutiev** [asweigart/pyscreeze#104]
I'm taking a screenshot of region but instead I have full desktop screenshot
```
import pyscreeze
import cv2
import numpy as np
import os
import pathlib


os.chdir(pathlib.Path(__file__).parent.resolve())
screenshot = pyscreeze.screenshot(region=(0,0, 50,50))
screenshot = cv2.cvtColor(np.array(screenshot), cv2.COLOR_RGB2BGR)
cv2.imwrite('captured-screen.png', screenshot)
```

my hardware setup:
macmini m2, one monitor

![image](https://github.com/asweigart/pyscreeze/assets/2060695/b33bda00-8cef-470b-842e-18ae98010683)

**Comment by adonishong** [comment]
same issue met here, 
macOS with M2 Max, Ventura 13.4.1
python environment is managed with anaconda, python 3.10, pyscreeze 0.1.29

**Comment by asweigart** [comment]
This fix is going into 1.0.0. Thanks!

**Comment by samik3301** [comment]
> This fix is going into 1.0.0. Thanks!

I was actually looking forward to make this as my first successful PR getting merged and an open source contribution but anyways, thanks for acknowledging and including it in the new version. Happy to help! :)
